docs/running-amplication-platform/configure-github-auth.md

```diff
@@ -26,14 +26,14 @@ Amplication leverage GitHub OAuth implementation that supports the standard [aut
 
 2. Enter the fields as follows:
    - **Application name** - `[your-github-username]-amplication-local`
-   - **Homepage URL** - `http://localhost:3001. _If you are hosting the Amplication client on any other address, use the specific address._
```

Comment:
@endalk200 please just add the '.' after the '`' and we're good to go. Thanks for contributing!